move/operators/sources/operators.move

```diff
@@ -113,24 +117,31 @@ module operators::operators {
 
         event::emit(CapabilityStored {
             cap_id,
+            cap_name: type_name::get<T>().into_string(),
         });
     }
 
     /// Allows an approved operator to borrow a capability by its ID.
-    public fun borrow_cap<T: key + store>(self: &Operators, operator_cap: &OperatorCap, cap_id: ID): &T {
-        let operator_id = object::id(operator_cap);
-
-        assert!(self.operators.contains(&operator_id), EOperatorNotFound);
+    public fun borrow_cap<T: key + store>(
+        self: &Operators,
+        _operator_cap: &OperatorCap,
```

Comment:
why keep this unused arg? you can use also index syntax by removing this perhaps

Reply:
This is what checks access through capability verification, just check on the `sender` is not enough. Because it's like `tx.origin` and not like `msg.sender`.